deid/dicom/parser.py

```diff
@@ -427,9 +427,8 @@ def _run_action(self, field, action, value=None):
             if value is not None:
 
                 # Jitter the field by the supplied value
-                jitter_timestamp(
-                    dicom=self.dicom, field=field.element.keyword, value=value
-                )
+                new_val = jitter_timestamp(field=field, value=value)
```

Comment:
Re my comment above: could field ever be a string here?

Reply:
See above.